guides/common/modules/proc_comparing-content-view-versions.adoc

```diff
@@ -4,7 +4,9 @@
 Use this procedure to compare Content View Version functionality for {Project}.
 
 .Procedure
-* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click *Compare*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
+. Select the content view version you want to compare.
```

Comment:
The **Click compare** part is now missing, but I assume that still needs to be clicked in some step?